### PR TITLE
AA-1638 Remove Product Improvement

### PR DESCRIPTION
**Pull request by lchavarriaga**

### CODE REVIEW

File: Application/EdFi.Ods.AdminApp.Web/EdFi.Ods.AdminApp.Web.csproj [hunk 1]
```diff
@@ -13,6 +13,8 @@
 
     <Content Include="Artifacts\**" CopyToPublishDirectory="Always" CopyToOutputDirectory="Never" />
     <Content Include="Schema/**/*.*" CopyToPublishDirectory="Always" />
+    <None Remove="Artifacts\MsSql\Structure\Admin\202301301600-RemoveProductImprovement.sql" />
+    <None Remove="Artifacts\PgSql\Structure\Admin\202301301600-RemoveProductImprovement.sql" />
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApp.Web/EdFi.Ods.AdminApp.Web.csproj:17]: @lchavarriaga 
Why are we removing the artifacts !?

File: Application/EdFi.Ods.AdminApp.Web/EdFi.Ods.AdminApp.Web.csproj [hunk 2]
```diff
@@ -54,4 +56,8 @@
     <Content Update="appsettings.SqlServer-YearSpecific.json" CopyToPublishDirectory="Never" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Folder Include="Views\Shared\Components\Tracker\" />
+  </ItemGroup>
+
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApp.Web/EdFi.Ods.AdminApp.Web.csproj:62]: @lchavarriaga 
Why folder needs to be included !?

File: Application/EdFi.Ods.AdminApp.Web/Views/Setup/PostUpdateSetup.cshtml [hunk 3]
```diff
@@ -41,15 +41,15 @@ See the LICENSE and NOTICES files in the project root for more information.
         $("#transient-error-note").hide();
         
         $.ajax({
-            url: "@Url.Action("CompletePostUpdateSetup")",
+			url: "@Url.Action("CompleteFirstTimeSetup")",
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApp.Web/Views/Setup/PostUpdateSetup.cshtml:44]: @lchavarriaga 
Please adjust the alignment on lines 44 and 52!!

File: Application/EdFi.Ods.AdminApp.Web/Artifacts/PgSql/Structure/Admin/202301301600-RemoveProductImprovement.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
+-- SPDX-License-Identifier: Apache-2.0
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApp.Web/Artifacts/PgSql/Structure/Admin/202301301600-RemoveProductImprovement.sql:1]: @lchavarriaga 
Have you tested these scripts on postgresql ?

File: Application/EdFi.Ods.AdminApp.Web/ActionFilters/SetupRequiredFilter.cs [hunk 5]
```diff
@@ -33,9 +33,12 @@ public override void OnActionExecuting(ActionExecutingContext filterContext)
...
 
-            filterContext.Result = OdsInstanceFirstTimeSetupCompleted()
-                ? RouteHelpers.RedirectToActionRoute<SetupController>(x => x.PostUpdateSetup())
-                : RouteHelpers.RedirectToActionRoute<SetupController>(x => x.FirstTimeSetup());
+            filterContext.Result = RouteHelpers.RedirectToActionRoute<SetupController>(x => x.PostUpdateSetup());
+
+
+            //filterContext.Result = OdsInstanceFirstTimeSetupCompleted()
+            //    ? RouteHelpers.RedirectToActionRoute<SetupController>(x => x.PostUpdateSetup())
+                //: RouteHelpers.RedirectToActionRoute<SetupController>(x => x.FirstTimeSetup());
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApp.Web/ActionFilters/SetupRequiredFilter.cs:41]: @lchavarriaga 
Please remove commented lines!

File: Application/EdFi.Ods.AdminApp.Management/Database/Models/ApplicationConfiguration.cs [hunk 6]
```diff
@@ -11,8 +11,8 @@ public class ApplicationConfiguration : Entity
 
         public bool FirstTimeSetUpCompleted { get; set; }
 
-        public bool EnableProductImprovement { get; set; }
+        //public bool EnableProductImprovement { get; set; }
 
-        public string ProductRegistrationId { get; set; }
+        //public string ProductRegistrationId { get; set; }
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApp.Management/Database/Models/ApplicationConfiguration.cs:16]: @lchavarriaga 
Please remove commented lines!

File: Application/EdFi.Ods.AdminApp.Management.Tests/Configuration/Configuration/ApplicationConfigurationServiceTests.cs [hunk 7]
```diff
@@ -48,142 +44,33 @@ public void ShouldAllowRegistrationByConfigurationAfterFirstRegistration()
...
             return Transaction(database =>
             {
-                bool allowUserRegistration = false;
+                var allowUserRegistration = false;
 
                 Scoped<AdminAppIdentityDbContext>(identity =>
                 {
-                    var appSettings = new AppSettings { EnableProductImprovementSettings = true };
-                    allowUserRegistration = new ApplicationConfigurationService(database, identity, Options.Create(appSettings)).AllowUserRegistration();
+                    //var appSettings = new AppSettings { EnableProductImprovementSettings = true };
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApp.Management.Tests/Configuration/Configuration/ApplicationConfigurationServiceTests.cs:58]: @lchavarriaga 
Please remove commented lines!

**Review by CSR2017 — COMMENTED** [review]
Will be doing functional review and testing

**Review by jasonh-edfi — APPROVED** [review]
Approved with one comment to remove additional config value, no longer needed

File: Application/EdFi.Ods.AdminApp.Web/Views/Setup/PostUpdateSetup.cshtml (hunk 3 above)
Comment (lchavarriaga) [reply Application/EdFi.Ods.AdminApp.Web/Views/Setup/PostUpdateSetup.cshtml:44]: They are aligned. What do you see?

File: Application/EdFi.Ods.AdminApp.Web/EdFi.Ods.AdminApp.Web.csproj (hunk 1 above)
Comment (lchavarriaga) [reply Application/EdFi.Ods.AdminApp.Web/EdFi.Ods.AdminApp.Web.csproj:17]: It seems those lines were added automatically by VS. I will remove them.

File: Application/EdFi.Ods.AdminApp.Web/EdFi.Ods.AdminApp.Web.csproj (hunk 2 above)
Comment (lchavarriaga) [reply Application/EdFi.Ods.AdminApp.Web/EdFi.Ods.AdminApp.Web.csproj:62]: Also added automatically by VS. Fixing.

File: Application/EdFi.Ods.AdminApp.Management.Tests/Configuration/Configuration/ApplicationConfigurationServiceTests.cs (hunk 7 above)
Comment (lchavarriaga) [reply Application/EdFi.Ods.AdminApp.Management.Tests/Configuration/Configuration/ApplicationConfigurationServiceTests.cs:58]: Removing.

File: Application/EdFi.Ods.AdminApp.Management/Database/Models/ApplicationConfiguration.cs (hunk 6 above)
Comment (lchavarriaga) [reply Application/EdFi.Ods.AdminApp.Management/Database/Models/ApplicationConfiguration.cs:16]: Removing.

File: Application/EdFi.Ods.AdminApp.Web/ActionFilters/SetupRequiredFilter.cs (hunk 5 above)
Comment (lchavarriaga) [reply Application/EdFi.Ods.AdminApp.Web/ActionFilters/SetupRequiredFilter.cs:41]: Removing.

File: Application/EdFi.Ods.AdminApp.Web/Artifacts/PgSql/Structure/Admin/202301301600-RemoveProductImprovement.sql (hunk 4 above)
Comment (lchavarriaga) [reply Application/EdFi.Ods.AdminApp.Web/Artifacts/PgSql/Structure/Admin/202301301600-RemoveProductImprovement.sql:1]: Yes. Columns removed without dependencies errors.

**Comment by CSR2017** [comment]
changes look good. Developer testing completed.

**Pull request merged**